lib/_http_incoming.js

```diff
@@ -315,6 +315,8 @@ IncomingMessage.prototype._dump = function _dump() {
   if (!this._dumped) {
     this._dumped = true;
     this.resume();
+    // Flush the existing buffered data
+    this._readableState._flush();
```

Comment:
I doubt this will go over well either since there is a recent movement to avoid reaching directly into `_*State` stream properties from outside streams.